lib/PuppeteerSharp/Page.cs

```diff
@@ -446,10 +446,13 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
 
         public async Task CloseAsync()
         {
-            await _client?.Connection?.SendAsync("Target.closeTarget", new
+            if (_client != null && !_client.Connection.IsClosed)
```

Comment:
could `_client.Connection` be `null`?

Reply:
It shouldn't be we could double check.

Reply:
Would this be clear to you?
`if (!(_client?.Connection?.IsClosed ?? true))`

Reply:
I was thinking of suggesting something like that

Reply:
Awesome!

Reply:
How about now?